entity-management-definitions/pom.xml

```diff
@@ -94,6 +94,17 @@
 			<artifactId>corelib-utils</artifactId>
 			<version>${version.corelib}</version>
 		</dependency>
+		<dependency>
+			<groupId>eu.europeana.corelib</groupId>
```

Comment:
we must not have dependency on corelib, please remove

Reply:
The dependency needs to be switched to API-Commons module for schema org, see https://europeana.atlassian.net/browse/EA-2642?focusedCommentId=87788

Reply:
I wrote you on Slack regarding this, we decided to keep the module in corelib.